torch/tensor.py

```diff
@@ -414,13 +414,13 @@ def __rfloordiv__(self, other):
     __ge__ = _C._TensorBase.ge
     __abs__ = _C._TensorBase.abs
 
-    def __std_mean__(self, dim=None, unbiased=True, keepdim=False):
+    def _std_mean(self, dim=None, unbiased=True, keepdim=False):
```

Comment:
can you just make this a native function?

Reply:
I changed autograd test, to be able to avoid fail when there is a method that implemented only in torch